CONTRIBUTING.md

```diff
@@ -179,16 +179,16 @@ Please follow all style guides to the best of your abilities.
 
 ### Git commit messages
 
-For all commit messages, please use a short phrase that describes the specific change.
-For example, “Add feature to check normalization method string” is much preferred to “change code”.
-When appropriate, reference issues (via `#` plus number) .
+Pycytominer uses [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard for commit messages to aid in automatic changelog generation.
+The easiest way to prepare commit messages that follow this standard is to use [commitizen](https://commitizen-tools.github.io/commitizen/), which comes pre-installed in the devcontainer.
+A pre-push commit hook will also run automatically to ensure that your commit messages follow this standard.
 
 ### Python style guide
 
 For python code style, we use [black](https://github.com/psf/black).
 Please use black before committing any code.
 We will not accept code contributions that do not use black.
-If you have set up your development environment using one of the dev container options specified above, the containers will install all required formatting tools, which will run automatically on any modified files before commits (using a tool called [pre-commit](https://pre-commit.com/)).
+If you have set up your development environment using one of the devcontainer options specified above, the containers will install all required formatting tools, which will run automatically on any modified files before commits (using a tool called [pre-commit](https://pre-commit.com/)).
```

Comment:
Suggestion for simplifications.
```suggestion
The devcontainer will install all required formatting tools.
Formatting checks in devcontainers run automatically on any modified files before commits (using a tool called [pre-commit](https://pre-commit.com/)).
```

Reply:
Accepted.

Reply:
Revised this language to be dev environment agnostic.